### PR TITLE
Crystopia Server Update and Fixes

### PR DESCRIPTION
**Pull request by tnsjesper**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/crystopia/background.png
Comment (DoJapHD) [inline minecraft_servers/crystopia/background.png:1]: Compress image

File: minecraft_servers/crystopia/manifest.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "server_name": "crystopia",
+  "nice_name": "Crystopia",
+  "direct_ip": "crystopia.net",
+  "server_wildcards": ["mc.crystopia.net"],
+  "social": {
+    "web": "https://crystopia.net",
```
Comment (DoJapHD) [inline minecraft_servers/crystopia/manifest.json:7]: Website not reachable

File: minecraft_servers/crystopia/manifest.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "web_shop": "https://shop.crystopia.net",
```
Comment (DoJapHD) [inline minecraft_servers/crystopia/manifest.json:8]: Website not reachable

File: minecraft_servers/crystopia/manifest.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "web_support": "https://crystopia.net/wiki",
```
Comment (DoJapHD) [inline minecraft_servers/crystopia/manifest.json:9]: Website not reachable

File: minecraft_servers/crystopia/manifest.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "twitter": "CrystopiaNet",
+    "instagram": "crystopianetwork",
+    "discord": "https://discord.gg/z92a4YzANd"
+  },
+
+  "discord": {
+    "server_id": 1084507325450162247,
+    "rename_to_minecraft_name": false
+  },
+  "yt_trailer": "https://www.youtube.com/watch?v=Gwa4PAA6YC8"
```
Comment (DoJapHD) [inline minecraft_servers/crystopia/manifest.json:19]: Change to embed id

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/crystopia/manifest.json (hunk 1 above)
Comment (tnsjesper) [reply minecraft_servers/crystopia/manifest.json:7]: Yes i have

File: minecraft_servers/crystopia/manifest.json (hunk 2 above)
Comment (tnsjesper) [reply minecraft_servers/crystopia/manifest.json:8]: now
... i have fix it

File: minecraft_servers/crystopia/manifest.json (hunk 1 above)
Comment (DoJapHD) [reply minecraft_servers/crystopia/manifest.json:7]: Still not reachable

File: minecraft_servers/crystopia/manifest.json (hunk 2 above)
Comment (DoJapHD) [reply minecraft_servers/crystopia/manifest.json:8]: Nope

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
No changes made; Reopen PR if still needed

**Pull request closed without merging**